### PR TITLE
Make sure the perf stubs have "coq" in their name

### PR DESCRIPTION
**Pull request by rlepigre**
As pointed out by @JasonGross, https://github.com/coq/coq/pull/18105 did not fix the problem appropriately.

The current approach does the trick, here is the relevant bit of the `coq-core.install` file:
```
stublibs: [
  "_build/install/default/lib/stublibs/dllcoqperf_stubs.so"
  "_build/install/default/lib/stublibs/dllcoqrun_stubs.so"
]
```

### CODE REVIEW

File: perf/perf.c
Comment (SkySkimmer) [inline perf/perf.c:1]: why revert?

Comment (rlepigre) [reply perf/perf.c:1]: Because that turned out to be entirely useless.

Comment (rlepigre) [reply perf/perf.c:1]: Also, I find it a bit silly for a file to be named `coq_...` in the Coq repo, especially if that is not needed.

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
@coqbot merge now

**Pull request merged**